### PR TITLE
BUGFIX: ONE-4190 fixed Angular 2+ documentation page

### PR DESCRIPTION
**Pull request by strouhaljan**
Updated versions of dependencies and note mentioning the need to add a snippet to `polyfills.ts` file in order to run the application. Tested and updated back to v6.3.0 as we support two last major versions.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/171/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/49/ : SUCCESS in 7s

File: docs/ht_use_react_components_in_angular_2.x.md [hunk 1]
```diff
@@ -41,15 +41,17 @@ To be able to use the GoodData.UI Visual Components in your Angular 2+ environm
...
 ```
 or
 ```bash
-yarn add uuid invariant react@15.6.2 react-dom@15.6.2 @gooddata/react-components
-yarn add @types/react @types/react-intl --dev
+yarn add uuid invariant react@^16.5.2 react-dom@^16.5.2 @gooddata/react-components rxjs-compat@6
+yarn add @types/react @types/react-intl@2.3.8 --dev
 ```
 
+**NOTE:** When using `Angular v6+`, be sure to add `(window as any).global = window;` snippet to `polyfills.ts` file due to missing `global`.
```
Comment (evgeniabogranskaya) [inline docs/ht_use_react_components_in_angular_2.x.md]: **NOTE:** When using Angular 6+, be sure to add the `(window as any).global = window;` snippet to the `polyfills.ts` file due to missing `global`.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/172/ : SUCCESS in 2m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/50/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/173/ : SUCCESS in 1m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/47/ : SUCCESS in 20s

**Pull request merged**